docs/user-guide/recipes.ipynb

```diff
@@ -0,0 +1,139 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d7608bb4-d7ce-4a36-8021-f1a52ad27ec5",
+   "metadata": {},
+   "source": [
+    "# Recipes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fce21908-20af-4580-afc7-08aaa152e828",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing import NewType\n",
+    "\n",
+    "import sciline"
```

Comment:
If you imagine adding more recipes, we should make each stand-alone, i.e., move imports into the concrete example.

Reply:
I figured that these two are needed everywhere. But I can change it.

Reply:
Yes, but it makes it easier to copy-and-paste a working example if we repeat it. Or to split the page later.

Reply:
Done